### PR TITLE
update canceled and postponed for event single

### PR DESCRIPTION
**Pull request by paulmskim**
Ticket: https://moderntribe.atlassian.net/browse/EXT-171

### CODE REVIEW

File: src/views/single/canceled-status.php [hunk 1]
```diff
@@ -11,35 +11,43 @@
...
 if ( 'canceled' !== $status ) {
 	return;
 }
 
+$canceled_reason = get_post_meta( $event->ID, Event_Meta::$key_status_canceled_reason, true );
+
+if ( $canceled_reason ) {
+	$label = _x( 'Canceled', 'Text to display canceled on event single if there is a canceled reason.', 'tribe-ext-events-control' );
+} else {
+	$label = sprintf( _x( 'This %s has been canceled', 'Text to display canceled on event single if no canceled reason.', 'tribe-ext-events-control' ), tribe_get_event_label_singular() );
```
Comment (jesseeproductions) [inline src/views/single/canceled-status.php:34]: I was thinking we might not want to include the %s in the translation, but then we have only `This` be a translated string, which makes it harder to translate too. I think we can stick with this.

**Review by jesseeproductions — COMMENTED** [review]
Please add a readme entry then all set.

File: src/views/single/canceled-status.php (hunk 1 above)
Comment (Camwyn) [inline src/views/single/canceled-status.php:34]:
```suggestion
        /* translators: single event label */
	$label = sprintf( _x( 'This %s has been canceled', 'Text to display canceled on event single if no canceled reason.', 'tribe-ext-events-control' ), tribe_get_event_label_singular() );
```

File: src/views/single/postponed-status.php [hunk 2]
```diff
@@ -20,26 +20,33 @@
...
 if ( 'postponed' !== $status ) {
 	return;
 }
 
+$postponed_reason = get_post_meta( $event->ID, Event_Meta::$key_status_postponed_reason, true );
+
+if ( $postponed_reason ) {
+	$label = _x( 'Postponed', 'Text to display postponed on event single if there is a postponed reason.', 'tribe-ext-events-control' );
+} else {
+	$label = sprintf( _x( 'This %s has been postponed', 'Text to display postponed on event single if no postponed reason.', 'tribe-ext-events-control' ), tribe_get_event_label_singular() );
```
Comment (Camwyn) [inline src/views/single/postponed-status.php:33]:
```suggestion
	/* translators: single event label */
	$label = sprintf( _x( 'This %s has been postponed', 'Text to display postponed on event single if no postponed reason.', 'tribe-ext-events-control' ), tribe_get_event_label_singular() );
```

**Review by Camwyn — CHANGES_REQUESTED** [review]
Either add a comment as in the suggestions - or better yet, add something to the context string to indicate what the `%s` represents.

**Comment by lucatume** [comment]
This PR is outdated and can be closed.

**Pull request closed without merging**